app/models/concerns/foreman/thread_session.rb

```diff
@@ -69,11 +69,17 @@ def current=(o)
             raise(ArgumentError, "Unable to set current User, expected class '#{self}', got #{o.inspect}")
           end
 
-          if o.is_a?(User)
-            Rails.logger.info "Current user: #{o.login} (#{o.admin? ? 'administrator' : 'regular user'})"
+          if Rails.logger.debug? && o.is_a?(User)
```

Comment:
This would still prevent anything showing in info. You probably want to remove the condition on `debug?`